frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueView.tsx

```diff
@@ -132,22 +132,27 @@ export function MissionQueueView({ refreshInterval }: RefreshProps) {
         })
     }, [missionQueue, ongoingMissions])
 
-    const missionQueueDisplay = missionQueue.map(function (mission, index) {
-        return <MissionQueueCard key={index} mission={mission} onDeleteMission={onDeleteMission} />
-    })
-
-    const loadingQueueDisplay = () => {
-        return <MissionQueueCard key={'test'} mission={dummyMission} onDeleteMission={() => {}} />
-    }
+    const missionQueueDisplay = missionQueue.map((mission, index) => (
+        <MissionQueueCard key={index} order={index + 1} mission={mission} onDeleteMission={onDeleteMission} />
+    ))
+
+    const loadingQueueDisplay = (
+        <MissionQueueCard
+            key={'test'}
+            order={missionQueue.length + 1}
+            mission={dummyMission}
+            onDeleteMission={() => {}}
+        />
+    )
 
     return (
         <StyledMissionView>
             <Typography variant="h1" color="resting">
                 {TranslateText('Mission Queue')}
             </Typography>
             <MissionTable>
-                {loadingMissionNames.size > 0 && loadingQueueDisplay()}
                 {missionQueue.length > 0 && missionQueueDisplay}
+                {loadingMissionNames.size > 0 && loadingQueueDisplay}
                 {loadingMissionNames.size === 0 && missionQueue.length === 0 && <EmptyMissionQueuePlaceholder />}
```

Comment:
```suggestion
               {loadingMissionNames.size > 0 
                ? loadingQueueDisplay() 
                : missionQueue.length === 0 && <EmptyMissionQueuePlaceholder />}
```

Reply:
I think the readability is better without this change